.circleci/config.yml

```diff
@@ -6,9 +6,11 @@ commands:
       - checkout
       - run:
           command: |
-            sudo pip install pyflakes==1.5.0 pylint setuptools-lint
+            sudo pip install pyflakes==1.5.0 pylint setuptools-lint coverage
             sudo pip install --upgrade setuptools
-            pyflakes beeline && python setup.py lint --lint-rcfile pylint.rc && python setup.py test
+            pyflakes beeline && python setup.py lint --lint-rcfile pylint.rc && coverage run ./setup.py test && coverage report --include="beeline/*" && coverage html --include="beeline/*"
```

Comment:
do we need to run all these together with `&&`? It makes it a little hard to read what is going on here. Presumably just having each on a separate line would do the same job?